R/zzz.R

```diff
@@ -74,3 +74,14 @@ clog <- function(x){
 pdf_parse <- function(x, parse) {
   if (parse) pdftools::pdf_text(x) else x
 }
+
+create_batch_query_list <- function(queries, page, pageSize) {
+  queryList <- lapply(queries, function(x){
+    as.list(setNames(x, rep("query", length(x))))
```

Comment:
namespace `setNames` here like `stats::setNames` since we don't import the stats pkg